plugins/become/ksusu.py

```diff
@@ -0,0 +1,182 @@
+# -*- coding: utf-8 -*-
+# Copyright (c) 2021, Ansible Project
+# GNU General Public License v3.0+ (see LICENSES/GPL-3.0-or-later.txt or https://www.gnu.org/licenses/gpl-3.0.txt)
+# SPDX-License-Identifier: GPL-3.0-or-later
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+DOCUMENTATION = """
+    name: ksusu
+    short_description: Kerberos substitute (ksu) user followed by substitute user (su)
+    description:
+        - This become plugins allows your remote/login user to execute commands as another user by calling C(su) through the C(ksu) utility.
+    author:
+        - Ed Schaller (@schallee)
+    version_added: 6.2.0
+    options:
+        become_user_ksu:
+            description: User you use with C(ksu) to run C(su).
+            default: root
+            ini:
+              - section: privilege_escalation
+                key: become_user_ksu
+              - section: ksusu_become_plugin
+                key: user_ksu
+            vars:
+              - name: ansible_become_user_ksu
+              - name: ansible_ksusu_user_ksu
+            env:
+              - name: ANSIBLE_BECOME_USER_KSU
+              - name: ANSIBLE_KSUSU_USER_KSU
+            required: False
+        become_user:
+            description: User you 'become' to execute the task.
+            default: root
+            ini:
+              - section: privilege_escalation
+                key: become_user
+              - section: privilege_escalation
+                key: become_user_su
+              - section: ksusu_become_plugin
+                key: user
+              - section: ksusu_become_plugin
+                key: user_su
+            vars:
+              - name: ansible_become_user
+              - name: ansible_become_user_su
+              - name: ansible_ksusu_user
+              - name: ansible_ksusu_user_su
+            env:
+              - name: ANSIBLE_BECOME_USER
+              - name: ANSIBLE_KSUSU_USER_SU
```

Comment:
I think the following should suffice:
```suggestion
              - section: privilege_escalation
                key: become_user
              - section: ksusu_become_plugin
                key: user_su
            vars:
              - name: ansible_become_user
              - name: ansible_ksusu_user_su
            env:
              - name: ANSIBLE_BECOME_USER
              - name: ANSIBLE_KSUSU_USER_SU
```